### PR TITLE
Feature: Add frontpage slider functionality, solves #162

### PR DESCRIPTION
**Pull request by abias**
This PR replaces PR #391 as I was not able to push review changes to Matthias' PR branch

### CODE REVIEW

File: lang/en/theme_boost_union.php [hunk 1]
```diff
@@ -746,6 +746,71 @@
 $string['tileordersetting_desc'] = 'With this setting, you define the order position of the advertisement tile {$a->no}. By default, the advertisement tiles are ordered from top to bottom and left to right like you see them on this settings page here. However, you can decide to assign another order position with this setting. If you assign the same order position to two or more advertisement tiles, they will be ordered again according to the order on this settings page.';
 $string['tiletitlesetting'] = 'Advertisement tile {$a->no} title';
 $string['tiletitlesetting_desc'] = 'Here, you enter the title which should be displayed in the advertisement tile {$a->no}. This is an optional setting, the advertisement tile will be shown even if you do not set a title.';
+// Settings: Slider tab.
+$string['slidertab'] = 'Slider';
+// ... Section: Slider general.
+$string['slidergeneralheading'] = 'Slider general';
+$string['slideranimationsetting'] = 'Slider animation type';
+$string['slideranimationsetting_desc'] = 'With this setting, you control the slider animation. \'Slide\' applies a sliding animation, \'fade\' applies a fading animation and \'none\' removes all animations.';
```
Comment (lucaboesch) [inline lang/en/theme_boost_union.php:754]: `$string['slideranimationsetting_desc'] = 'With this setting, you control the slider animation. \'Slide\' applies a sliding animation, \'Fade\' applies a fading animation and \'None\' removes all animations.';`

Set upper case as in the select, please.

File: lang/en/theme_boost_union.php [hunk 2]
```diff
@@ -746,6 +746,71 @@
...
+$string['sliderintervalsetting_desc'] = 'With this setting, you control how long a slide is displayed in milliseconds. The minimum value is 1000 (one second) and the maximum value is 10000 (10 seconds).';
+$string['sliderkeyboardsetting'] = 'Allow slider keyboard interaction';
+$string['sliderkeyboardsetting_desc'] = 'With this setting, you enable keyboard inputs (arrow keys) to control the slider. Please note that disabling this lowers accessibility.';
+$string['sliderpausesetting'] = 'Pause slider on mouseover';
+$string['sliderpausesetting_desc'] = 'With this setting, you prevent the slider from cycling through the slides when a user hovers over a slide. Please note that disabling this lowers accessibility.';
+$string['sliderridesetting'] = 'Cycle through slides';
+$string['sliderridesetting_desc'] = 'With this setting, you control the cycling behaviour of the slider. \'On page load\' begins cycling through slides after the page has finished loading, \'after interaction\' will start cycling after a user has interacted with the slider. \'Never\' disables the automatic cycling of slides altogether, requiring user input to cycle through slides.';
```
Comment (lucaboesch) [inline lang/en/theme_boost_union.php:775]: `$string['sliderridesetting_desc'] = 'With this setting, you control the cycling behaviour of the slider. \'On page load\' begins cycling through slides after the page has finished loading, \'After interaction\' will start cycling after a user has interacted with the slider. \'None\' disables the automatic cycling of slides altogether, requiring user input to cycle through slides.';`

Upper case as in the select.
Also, there is no 'Never'. Either change here to 'None' or change to 'Never' further down.

File: lang/en/theme_boost_union.php [hunk 3]
```diff
@@ -746,6 +746,71 @@
...
+$string['sliderridesetting_afterinteraction'] = 'After interaction';
+$string['sliderridesetting_none'] = 'None';
```
Comment (lucaboesch) [inline lang/en/theme_boost_union.php:777]: Or 'Never', as remarked further up.

**Review by lucaboesch — DISMISSED** [review]
I found some minor things but other than that, this is good to go.
Please merge, @abias.

**Comment by abias** [comment]
Thank you very much, Luca, for having a look at this patch.
I have fixed the glitches which you mentioned and will merge the PR now.

**Pull request merged**